packages/terra-form-checkbox/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Unreleased
 * Updated to use React 16
 * Update theming values to initial browser default for various properties
 * Merged border-color, border-width, and border-style into short-hand syntax
+* Removed erroneous default margin-right value of 2rem
```

Comment:
You might update this description to say `reduced` instead of `removed` or something similar

Reply:
Good call! 23e96eab